tee-worker/omni-executor/rpc-server/src/methods/omni/get_omni_account.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2024 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::ErrorCode;
+use crate::{methods::omni::common::PumpxRpcError, server::RpcContext};
+use executor_primitives::{utils::hex::ToHexPrefixed, Web2IdentityType};
+use heima_primitives::Identity;
+use jsonrpsee::{types::ErrorObject, RpcModule};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct GetOmniAccountParams {
+	pub user_email: String,
```

Comment:
We should probably check in all methods that the email address is valid (Not in this PR)